Action.c

```diff
@@ -714,4 +756,6 @@ void Action_setBindings(Htop_Action* keys) {
    keys[KEY_F(10)] = actionQuit;
    keys[KEY_F(18)] = actionExpandCollapseOrSortColumn;
    keys[KEY_RECLICK] = actionExpandOrCollapse;
+   keys[KEY_SHIFT_TAB] = actionPrevScreen;
```

Comment:
This key is Ctrl+F9
See [[Function Keys]](http://bio.gsi.de/DOCS/www.coredumps.de/doc/ibm/aix/aix_hints_and_tips/3014666.htm#4)